x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/index.test.tsx

```diff
@@ -219,7 +219,9 @@ describe('unified timeline', () => {
     );
   });
 
-  describe('columns', () => {
+  // Flaky : See https://github.com/elastic/kibana/issues/179831
+  // removing/moving column current leads to infitinite loop, will be fixed in further PRs.
+  describe.skip('columns', () => {
```

Comment:
just out of curiosity, I can reproduce the infinite loop on `main` right now, so why skipping this test in this PR? Have the changes introduced here made it worse?

Reply:
This whole suite is skipped in the main but I was trying to enable some tests and skip some but in the end I found out the whole suite is currently un-stable because of that issue. 

This change does not impact anything since container suite is already skipped.